A32NX/de-DE.locPak

```diff
@@ -14,7 +14,7 @@
       "A32NX.LOADING.TIPS.A320NEO_003": "FlyByWire ist nach der Fly-by-Wire-Steuerung benannt, die erstmals im Airbus A320 eingeführt wurde.",
       "A32NX.LOADING.TIPS.A320NEO_004": "Es sind 9693 Flugzeuge der A320-Familie gebaut worden! Das ist fast so viel wie die Bevölkerung von Tuvalu!",
       "A32NX.LOADING.TIPS.A320NEO_005": "Airbus A320neo Flugzeuge haben eine Reichweite von 3500 NM oder 6500 KM. Das ist genug, um von New York nach Paris zu fliegen!",
-      "A32NX.LOADING.TIPS.A320NEO_006": "Wussten Sie schon? Der A320-Familie hat 4 Varianten! Der A318 A319 A320 und A321."
+      "A32NX.LOADING.TIPS.A320NEO_006": "Wussten Sie schon? Die A320-Familie hat 4 Varianten! Den A318 A319 A320 und A321."
```

Comment:
```suggestion
      "A32NX.LOADING.TIPS.A320NEO_006": "Wussten Sie schon? Die A320-Familie hat 4 Varianten! Den A318, A319, A320, und A321."
```

Not sure if this applies in German but wouldn't you comma separate these items?

Reply:
good suggestion. makes sense ofc